types/victory/index.d.ts

```diff
@@ -5,6 +5,8 @@
 //                 Krzysztof Cebula <https://github.com/Havret>
 //                 Vitaliy Polyanskiy <https://github.com/allreadyExisted>
 //                 James Lismore <https://github.com/jlismore>
+//                 Stack Builders <https://github.com/stackbuilders>
+//                 Esteban Ibarra <https://github.com/ibarrae>
```

Comment:
Who are Stack Builders. Is this your company? I apologize for this question.

Reply:
Yeah, its also the fork from the changes are being made.